app/controllers/restaurants_controller.rb

```diff
@@ -1,7 +1,6 @@
 class RestaurantsController < ApplicationController
   def index
-    restaurants = Restaurant.all
-    render json: restaurants.as_json(include: [:addresses, :reviews])
+    render json: Restaurant.all.as_json(include: [:addresses, :reviews, :genres])
```

Comment:
When you eject `.as_json` you might be giving away information you don't want to, like `id` and other attributes the client should not know about (in real life it could be users' email, address etc). If you haven't already, move to serializers.

Reply:
I think that in simple cases like this one, I would use the ```except``` or ```only``` as part of the ```as_json``` as serializers can be "hidden" and in this case it is explicit. I think I would use serializer for pure API. WDYT?